frontend/src/metabase/public/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { DEFAULT_DASHBOARD_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
+import type { EmbedDisplayParams } from "metabase/dashboard/types";
+
+export const DEFAULT_EMBED_DISPLAY_PARAMS: Partial<EmbedDisplayParams> = {
+  titled: DEFAULT_DASHBOARD_DISPLAY_OPTIONS.titled,
```

Comment:
By using `DEFAULT_DASHBOARD_DISPLAY_OPTIONS` we display the relationship between the 2 constants. In this case, we could see that:
- We have fewer properties here
- `theme` by default is `undefined`. That means we don't show `theme`, but it means the light theme under the hood. This is to preserve all existing behavior.

It might make sense to refactor `theme` to be `"light"` just to make it consistent with the other constant, but it is out of scope for this refactor.